### DISCUSSION TITLE
Animations with a decimal frame ms time are not displayed in a storyboard.

### DISCUSSION THREAD

**Discussion by Coppertine** [ppy/osu#18240] [Q&A] [answered]
I came to this issue when replicating the IntroTriangles scene into a stable storyboard.
I got the frac shaders exported into individual png files and had them placed into an animation (using a duration / frame count formula).
The following storyboard code shown below
```
Animation,Background,Centre,"sb\ani\lazer\.png",320,240,24,38.3333333333333,LoopOnce
 F,0,2080,,1
 S,0,2080,2723,1.5,1.3
 S,12,2723,3000,1.3,0.4
 F,0,3000,,0
 ```
 Beatmap: https://osu.ppy.sh/beatmapsets/1518063#osu/3117135
 
As noted, the frame duration is decimal value, my guess would be that lazer would only accept integer millisecond values whilst stable would either round them or utilise them fully.

Stable Editor with animation in view
![image](https://user-images.githubusercontent.com/37494321/168071819-ef4161f0-138a-469b-a52f-56bb72f97b29.png)

Lazer gameplay screenshoted at the time the animation is intended to appear
![osu_2022-05-12_22-07-41](https://user-images.githubusercontent.com/37494321/168071910-a44175dd-8f99-4c30-8195-bb8473c8a7ba.png)

Log:
[runtime.log](https://github.com/ppy/osu/files/8678882/runtime.log)
[database.log](https://github.com/ppy/osu/files/8678909/database.log)
[performance.log](https://github.com/ppy/osu/files/8678910/performance.log)
[network.log](https://github.com/ppy/osu/files/8678911/network.log)

**Comment by peppy** [comment, marked as answer]
I'd say it's more likely to be the broken filename?

**Reply by Coppertine** [reply]
it could be, although it is funcitonal on stable without issues as `/.png` is valid for an animation path since it would just use `/0.png` - `/23.png` instead.

**Reply by peppy** [reply]
how about you don't

**Reply by Coppertine** [reply]
I might have to remind you that there could be ranked storyboards utilising this method of naming.
